### PR TITLE
Updated cluster creation tutorial

### PR DESCRIPTION
**Pull request by n-boshnakov**
**How to categorize this PR?**

/area documentation
/kind enhancement
/platform alicloud

**What this PR does / why we need it**:
This PR updates the outdated tutorial for creating a Kubernetes cluster on Alibaba Cloud.
**Which issue(s) this PR fixes**:
Part of [gardener/documentation#287](https://github.com/gardener/documentation/issues/287).

**Special notes for your reviewer**:

**Release note**:

```doc developer
NONE
```

### CODE REVIEW

**Review by jia-jerry — DISMISSED** [review]
/lgtm

File: docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+     1. Paste the policy that you copied from the Gardener dashboard to this custom policy.
+     1. Choose *OK*.
+
+          <img src="images/alicloud-paste-policy.png">
+
+1. In the Alibaba Cloud console, create a new technical user:
+     1. Choose *RAM* \> *Users*.
+     1. Choose *Create User*.
+
+          <img src="images/gardener-create-user.png">
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md:49]: Can you hide the names of existing users in this screenshot?

File: docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+     > After the user is created, `AccessKeyId` and `AccessKeySecret` are generated and displayed. Remember to save them. The `AccessKey` is used later to create secrets for Gardener.
+
+     <img src="images/gardener-user-created.png">
+      
+1. Assign the policy you created to the technical user:
+     1. Choose *RAM* \> *Permissions* \> *Grants*.
+     1. Choose *Grant Permission*.
+
+          <img src="images/grant-permission.png">
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md:65]: Can you hide the principals names in this screenshot?

File: docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+     1. Select *Alibaba Cloud Account*. 
+     1. Assign the policy you’ve created before to the technical user.
+
+          <img src="images/assign-policy.png">
```
Comment (ialidzhikov) [inline docs/tutorials/kubernetes-cluster-on-alicloud-with-gardener/_index.md:70]: Can you hide the principal name in this screenshot?

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**